src/dos/dos.cpp

```diff
@@ -49,114 +49,143 @@ void DOS_SetError(Bit16u code) {
 	dos.errorcode=code;
 }
 
-std::array<uint16_t, 1> countries_date_format_mmddyyyy = {
-        COUNTRY::United_States,
-};
-
-std::array<uint16_t, 12> countries_date_format_yyyymmdd = {
-        COUNTRY::Candian_French, COUNTRY::Hungary,      COUNTRY::Yugoslavia,
-        COUNTRY::Romania,        COUNTRY::Czech_Slovak, COUNTRY::Sweden,
-        COUNTRY::Poland,         COUNTRY::Japan,        COUNTRY::South_Korea,
-        COUNTRY::China,          COUNTRY::Iceland,      COUNTRY::Taiwan,
-};
-
-std::array<uint16_t, 11> countries_date_separator_slash = {
-        COUNTRY::Latin_America, COUNTRY::Greece, COUNTRY::Belgium,
-        COUNTRY::Spain,         COUNTRY::Italy,  COUNTRY::United_Kingdom,
-        COUNTRY::Brazil,        COUNTRY::Turkey, COUNTRY::Arabic,
-        COUNTRY::Taiwan,        COUNTRY::Israel,
-};
-
-std::array<uint16_t, 8> countries_date_separator_period = {
-        COUNTRY::Russia,  COUNTRY::France,  COUNTRY::Switzerland,
-        COUNTRY::Austria, COUNTRY::Norway,  COUNTRY::Germany,
-        COUNTRY::China,   COUNTRY::Finland,
-};
-
-std::array<uint16_t, 3> countries_time_12hour_clock = {
-        COUNTRY::United_States,
-        COUNTRY::Latin_America,
-        COUNTRY::Australia,
-};
+static constexpr Bit8u DATE_MDY = 0;
+static constexpr Bit8u DATE_DMY = 1;
+static constexpr Bit8u DATE_YMD = 2;
```

Comment:
This is looking top-notch @FeralChild64 !

super minor comment: Staging and -X have both deprecated DOSBox's custom types (`Bitu8u`, `Bitu`, etc.. ) for the usual C99 `#include <stdint>` types.. `uint8_t`, etc.